### ISSUE TITLE
32 bit CI Failures

### ISSUE THREAD

**Issue by WillAyd** [pandas-dev/pandas#32229]
Haven't had time to debug but seeing this in the logs when setting up the 32 bit linux environment:

```sh
Traceback (most recent call last):
  File "/home/vsts/miniconda3/bin/conda", line 7, in <module>
    from conda.cli import main
ImportError: No module named conda.cli
```

**Comment by jreback** [comment]
just disable the build for now (and leave this issue open)

**Comment by simonjayhawkins** [comment]
closed in #32241
